src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -579,7 +622,7 @@ public TimeSpan GetTimeoutOrDefault(TimeSpan? timeout)
         public IActorRef ChildActorOf(
             Props props, SupervisorStrategy supervisorStrategy, CancellationToken cancellationToken = default)
             => ChildActorOfAsync(props, supervisorStrategy, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.